nextflow.config

```diff
@@ -46,8 +45,8 @@ params {
     taxa_classification_mmseqs_taxonomy_searchtype   = 2
     taxa_classification_mmseqs_taxonomy_lcaranks     = 'kingdom,phylum,class,order,family,genus,species'
     taxa_classification_mmseqs_taxonomy_taxlineage   = 1
-    taxa_classification_mmseqs_taxonomy_sensitivity  = '5.0'
-    taxa_classification_mmseqs_taxonomy_orffilters   = '2.0'
+    taxa_classification_mmseqs_taxonomy_sensitivity  = 5.0
+    taxa_classification_mmseqs_taxonomy_orffilters   = 2.0
```

Comment:
Did you test this? Referring to the issue of floats getting reformatted to scientific or int notation that we had once.

Reply:
No, can't test at the moment beause of the mmseqs problems :grimacing:

Reply:
And I think tis was from the auto-formatting agian..